spec/integration/cli_instances_spec.rb

```diff
@@ -50,12 +48,10 @@
         |   process-1 | running |               |             |
         |   process-2 | running |               |             |
         |   process-3 | failing |               |             |
-        +-------------+---------+---------------+-------------+
```

Comment:
why was the separator removed? we still want to the separator between instances if --ps is specified.